CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # HEAD
 
 ## Features
+* **component:** Remove deprecated Hero component. (#912)
```

Comment:
We should also add a line to the migration tips below to convert any uses of Hero to either Split or Callout.